python/data_structures/linked_list.py

```diff
@@ -1,3 +1,7 @@
+from optparse import Values
+from turtle import setundobuffer
```

Comment:
looks like your editor auto-imported a couple of things that you're not using. Be sure to keep an eye out for this -- I actually have a very similar problem and haven't really found a good fix yet.